tasks/install-package.yml

```
@@ -0,0 +1,114 @@
+- name: install the rpm repo's key
+  rpm_key:
+    state: present
+    key: "{{ rpm_repo_key }}"
+  when: ansible_os_family == "RedHat"
+
+- name: install apt repo keys
+  apt_key:
+    state: present
+    url: "{{ item.url }}"
+    id: "{{ item.id }}"
+  when: ansible_os_family == "Debian"
+  loop:
+  - { url: "{{ deb_repo_key }}", id: "{{ deb_repo_key_id }}" }
+  - { url: "https://deb.nodesource.com/gpgkey/nodesource.gpg.key", id: "1655A0AB68576280" }
+
+- name: install the rpm repo
+  package:
+    name: "{{ rpm_repo_url }}"
+    state: present
+    disable_gpg_check: "{{ disable_gpg_check_rpm_repo | bool }}"
+  when: ansible_os_family == "RedHat"
+
+- name: install the apt repo
+  apt:
+    deb: "{{ apt_repo_url }}"
+    state: present
+    update_cache: "{{ apt_update_cache | bool }}"
+  when: ansible_os_family == "Debian"
+
+- name: remove node 14 repo if it exists
+  file:
+    path: /etc/apt/sources.list.d/nodesource.list
+    state: absent
+  when: ansible_os_family == "Debian"
+
+- name: install node's apt repo
+  apt_repository:
+    repo: "{{ item }}"
+    state: present
+    filename: nodesource
+    update_cache: true
+  loop:
+    - "deb https://deb.nodesource.com/node_12.x {{ ansible_distribution_release }} main"
+    - "deb-src https://deb.nodesource.com/node_12.x {{ ansible_distribution_release }} main"
+  when: ansible_os_family == "Debian"
+
+- name: DELME HACK FOR DEBIAN TESTING
+  shell: sed -i 's#latest#build/2.0#g' /etc/apt/sources.list.d/ondemand-web.list
```

```suggestion
  shell: sed -i 's#ondemand/2.0/web#ondemand/build/2.0/web#g' /etc/apt/sources.list.d/ondemand-web.list
```
The RPM repo file you install is 2.0 one, so won't have latest.  Also I found from E2E tests it's best to have larger part of URL to allow re-run not replacing `build/2.0` with `build/build/2.0`.